### PR TITLE
Add token to bridge

### PR DESCRIPTION
**Pull request by krofax**
Wrote a paragraph on how to add tokens to the bridge.

### CODE REVIEW

File: docs/dev/README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ If this is your first time using zkSync, we recommend that you kick off at the b
 - [Account abstraction](./developer-guides/aa.md) - Learn about account abstraction.
 - [Fee mechanism](./developer-guides/transactions/fee-model.md) - A quick overview of the zkSync fee structure.
 - [Bridging of funds](./developer-guides/bridging/bridging-asset.md) - A brief intro on token bridging.
+    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-bridge) - Add your token to the bridge
```
Comment (kailash-manasarovar) [inline docs/dev/README.md:25]:
```suggestion
    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-bridge) - Add your token to the bridge.
```

File: docs/dev/developer-guides/bridging/bridging-asset.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ Addresses of tokens on L2 will always differ from the same token L1 address.
 
 :::
 
+## Add tokens to bridge
+
+To add your token to the bridge, fill and submit the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw) and we will get your token added to the bridge.
+
```
Comment (kailash-manasarovar) [inline docs/dev/developer-guides/bridging/bridging-asset.md:21]:
```suggestion
## Add tokens to bridge

To add your token to the bridge, fill out and submit the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw) and we will get your token added to the bridge.

```

File: docs/dev/README.md (hunk 1 above)
Comment (kailash-manasarovar) [reply docs/dev/README.md]: Did you see Ramon's comment? https://most.matter-labs.io/matterlabs/pl/4fr3n3eieffdped5tjbphgpnwh

File: docs/dev/developer-guides/bridging/bridging-asset.md [hunk 3]
```diff
@@ -15,14 +15,14 @@ Addresses of tokens on L2 will always differ from the same token L1 address.
...
-
-To add your token to the bridge, fill and submit the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw) and we will get your token added to the bridge.
-
 ## Default bridges
 
 You can get default bridges' addresses using the `zks_getBridgeContracts` endpoint or `getDefaultBridgeAddresses` method of `Provider` in our [Javascript SDK](../../../api/js/) (similar methods are available in the other SDKs).
 
+### How to add tokens to the default bridge
+
+While our standard bridge can be used without requiring permission from a smart contract perspective, our bridge UI only displays tokens that are currently added to our SDK. If you would like to see a particular token added, you can submit a request by filling out the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw). Our team will review your request, and get back to you if we have any outstanding questions before adding the token to our list.
```
Comment (kailash-manasarovar) [inline docs/dev/developer-guides/bridging/bridging-asset.md:24]:
```suggestion
```

File: docs/dev/developer-guides/bridging/bridging-asset.md [hunk 4]
```diff
@@ -15,14 +15,14 @@ Addresses of tokens on L2 will always differ from the same token L1 address.
 
 :::
 
-## Add tokens to bridge
-
-To add your token to the bridge, fill and submit the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw) and we will get your token added to the bridge.
-
 ## Default bridges
```
Comment (kailash-manasarovar) [inline docs/dev/developer-guides/bridging/bridging-asset.md]:
```suggestion
## Add tokens to bridge

While the standard bridge can be used without permission from a smart contract perspective, the UI only displays tokens that have been added to our SDK. 

If you would like to add a token, submit a request by filling out the [token request form](https://5p68rkvrcqg.typeform.com/to/NbYpe2pw). Our team will review your request, and get back to you if we have any outstanding questions before adding the token to our list.

## Default bridges
```

## Default bridges
```

File: docs/dev/README.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ If this is your first time using zkSync, we recommend that you kick off at the b
 - [Account abstraction](./developer-guides/aa.md) - Learn about account abstraction.
 - [Fee mechanism](./developer-guides/transactions/fee-model.md) - A quick overview of the zkSync fee structure.
 - [Bridging of funds](./developer-guides/bridging/bridging-asset.md) - A brief intro on token bridging.
-    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-bridge) - Add your token to the bridge
+    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#how-to-add-tokens-to-the-default-bridge) - Add your token to the bridge.
```
Comment (kailash-manasarovar) [inline docs/dev/README.md:25]:
```suggestion
    - [Add tokens to bridge](./developer-guides/bridging/bridging-asset.md#add-tokens-to-the-bridge) - Add your token to the bridge.
```

File: docs/dev/README.md (hunk 1 above)
Comment (kailash-manasarovar) [reply docs/dev/README.md]: Hi, I made another change. I don't like the word Default either. It's confusing. What is a Default bridge? Is it the same as the standard bridge, I suspect so. If it is, I would change the title **Default bridge** to **Bridge addresses** which is clearer. I'll add that too now.

Comment (krofax) [reply docs/dev/README.md]: I've updated it @kailash-manasarovar

**Review by kailash-manasarovar — APPROVED** [review]

**Pull request merged**